DataProducts/draft/Employment/EscoOccupations.json

```diff
@@ -0,0 +1,3225 @@
+          "2611.1.1",
+          "2511.9",
+          "3521.2",
+          "2133.14",
+          "2330.1.4",
+          "2635.3.5",
+          "3322.1.2",
+          "7512.5",
+          "3141.2.5",
+          "8131.4",
+          "6223.2.1",
+          "2120.2",
+          "9612.2",
+          "5414.1.2",
+          "2422.1",
+          "8211.5"
+        ],
+        "type": "string",
+        "description": "An enumeration."
+      },
+      "EscoOccupationRequest": {
+        "title": "EscoOccupationRequest",
+        "type": "object",
+        "properties": {
+          "limit": {
+            "title": "Limit",
+            "minimum": 1.0,
+            "type": "integer",
+            "description": "The limit of items per page in response",
+            "example": 1000
+          },
+          "offset": {
+            "title": "Offset",
+            "minimum": 0.0,
+            "type": "integer",
+            "description": "The starting index of responses",
+            "example": 0
+          }
+        }
+      },
+      "EscoOccupationResponse": {
+        "title": "EscoOccupationResponse",
+        "required": ["occupations", "totalCount"],
+        "type": "object",
+        "properties": {
+          "occupations": {
+            "title": "Occupations",
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Occupation"
+            },
+            "description": "A list of relevant occupation classes based on ESCO standard"
+          },
+          "totalCount": {
+            "title": "Total Count",
+            "type": "integer",
+            "description": "The total count of occupation entries",
+            "example": 3568
+          }
+        }
+      },
+      "HTTPValidationError": {
+        "title": "HTTPValidationError",
+        "type": "object",
+        "properties": {
+          "detail": {
+            "title": "Detail",
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/ValidationError"
+            }
+          }
+        }
+      },
+      "Occupation": {
+        "title": "Occupation",
+        "required": ["escoCode", "escoJobTitle", "escoDescription", "escoIdentifier"],
+        "type": "object",
+        "properties": {
+          "escoCode": {
+            "title": "ESCO codes",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/EscoCode"
+              }
+            ],
+            "description": "The ESCO code based on the level 5 or narrower in ESCO classification",
+            "example": "8331.1"
+          },
+          "escoJobTitle": {
+            "title": "ESCO Job Title",
+            "maxLength": 40,
+            "type": "string",
+            "description": "The job title based on ESCO classification",
+            "example": "bus driver"
+          },
+          "escoDescription": {
+            "title": "ESCO Job Title Description",
+            "maxLength": 250,
```

Comment:
Let's remove the max length validation from ESCO description. They can occationally be rather long > 1000 characters.